tdrs-backend/tdpservice/users/test/test_auth_check.py

```diff
@@ -6,10 +6,11 @@
 
 
 @pytest.mark.django_db
-def test_auth_check_endpoint_with_no_user(api_client, user):
-    """If there is no user auth_check should return FORBIDDEN."""
+def test_auth_check_endpoint_with_no_user(api_client):
+    """If there is no user auth_check should return 200 with unauthorized message."""
     response = api_client.get(reverse("authorization-check"))
-    assert response.status_code == status.HTTP_403_FORBIDDEN
```

Comment:
The permissions change mentioned above allowed this request to go through without getting a 403. The test also ensures the user receives a false response on "authenticated"

Reply:
This is from the `AllowAny` change. If you look at the code, it is utilizing the HTTP status codes properly. This endpoint is set to allow any request and will return a message indicating whether the user is authenticated. The `403` response indicates the user does not have permission to hit the endpoint. This isn't the case. The user is allowed to hit the endpoint. However, it will provide the appropriate authorization information in the response.

This is exactly why the `AllowAny` change was made. So this would respond properly.